### PR TITLE
Upgrade Catalog Applications with UI and CLI

### PR DESCRIPTION
**Pull request by Colin1841**
Jira: https://jira.d2iq.com/browse/D2IQ-77905

We have the UI steps added here, but NOT yet the CLI steps. We can merge this and continue the CLI work next sprint, if necessary, or we can knock it out today and tomorrow if we have that procedure worked out.

ALSO, I started the new 2.2 release notes with a blurb about this.

### CODE REVIEW

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 1]
```diff
@@ -24,17 +24,14 @@ This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster
 
 ### New features and capabilities
 
-#### Feature 1, 2, etc.
+#### Upgrade catalog application via CLI and UI
 
+You can use either the CLI or the UI to [upgrade your catalog applications](..\projects\applications\catalog-applications\index.md).
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/release-notes/index.md:29]: Let's add a note somewhere in the release notes that catalog apps **must** be upgraded to the latest version BEFORE upgrading the kubernetes version (or konvoy version for managed konvoy clusters) on attached clusters, due to the previous catalog app versions incompatibility with k8s 1.22

Comment (cbuto) [inline pages/dkp/kommander/2.2/release-notes/index.md:29]:
```suggestion
You can use either the CLI or the UI to [upgrade your catalog applications](/projects/applications/catalog-applications/#upgrade-catalog-applications).
```

Comment (Colin1841) [reply pages/dkp/kommander/2.2/release-notes/index.md:29]: Thanks Casey, 101 ways to do the links always throws me off. I thought the backwards slashed looked weird - FWIW, that's how they copied down directly from Git, as a relative link!

Comment (Colin1841) [reply pages/dkp/kommander/2.2/release-notes/index.md:29]: Thanks Grace, I'll add this to that new section of the RN now.

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md [hunk 2]
```diff
@@ -5,5 +5,22 @@ title: Catalog applications
 menuWeight: 20
 excerpt: Catalog applications
 ---
-
 Catalog applications are any third-party or open source applications that appear in the Catalog. These can be DKP applications provided by D2iQ for use in your environment, or [Custom Applications](custom-applications) that can be used but are not supported by D2iQ.
+
+## Upgrade catalog applications
+
+Before upgrading your catalog applications, verify the current and supported versions of the application:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md:12]:
```suggestion
<p class="message--note"><strong>NOTE: Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous catalog application versions incompatibility with Kubernetes version 1.22.</p>

Before upgrading your catalog applications, verify the current and supported versions of the application:
```

File: pages/dkp/kommander/2.2/release-notes/index.md (hunk 1 above)
Comment (Colin1841) [inline pages/dkp/kommander/2.2/release-notes/index.md:29]:
```suggestion
You can use either the CLI or the UI to [upgrade your catalog applications](..\projects\applications\catalog-applications\index.md).

<p class="message--note"><strong>NOTE: Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous catalog application versions incompatibility with Kubernetes version 1.22.</p>
```

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 3]
```diff
@@ -24,17 +24,14 @@ This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster
 
 ### New features and capabilities
 
-#### Feature 1, 2, etc.
+#### Upgrade catalog application via CLI and UI
 
+You can use either the CLI or the UI to [upgrade your catalog applications](/projects/applications/catalog-applications/#upgrade-catalog-applications).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/release-notes/index.md:29]:
```suggestion
You can use either the CLI or the UI to [upgrade your catalog applications](/projects/applications/catalog-applications/#upgrade-catalog-applications).

<p class="message--note"><strong>NOTE: Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous versions' incompatibility with Kubernetes 1.22.</p>
```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 4]
```diff
@@ -24,17 +24,16 @@ This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster
...
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/release-notes/index.md:29]:
```suggestion
You can use either the CLI or the UI to [upgrade your catalog applications](../projects/applications/catalog-applications/#upgrade-catalog-applications).
```

**Review by cbuto — APPROVED** [review]

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 5]
```diff
@@ -24,17 +24,16 @@ This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster
...
 ### New features and capabilities
 
-#### Feature 1, 2, etc.
+#### Upgrade catalog application via CLI and UI
 
+You can use either the CLI or the UI to [upgrade your catalog applications](../projects/applications/catalog-applications/#upgrade-catalog-applications).
 
-### Component updates
-
+<p class="message--note"><strong>NOTE: Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous versions' incompatibility with Kubernetes 1.22.</p>
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/release-notes/index.md:31]:
```suggestion
<p class="message--note"><strong>NOTE:</strong> Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous versions' incompatibility with Kubernetes 1.22.</p>
```

**Pull request merged**